### PR TITLE
Add slack.spack.io to spack-infrastructure

### PR DESCRIPTION
**Pull request by tgamblin**
This is a draft deployment attempting to move the [slackin](https://github.com/emedvedev/slackin-extended) app hosted at https://spackpm.herokuapp.com to https://slack.spack.io.

### CODE REVIEW

File: kubernetes/spack/slack-spack-io/web/deployments.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+        image: "spack/slackin-extended:latest"
+        imagePullPolicy: Always
+        ports:
+        - name: http
+          containerPort: 80
+        env:
+        - name: SLACK_SUBDOMAIN
+          value: spackpm
+        - name: SLACKIN_ANALYTICS
+          value: "UA-101208306-3"
```
Comment (opadron) [inline kubernetes/spack/slack-spack-io/web/deployments.yaml:33]: Is this sensitive?  If so, you should move it to your secret.

**Review by opadron — COMMENTED** [review]
Looking good so far.  Let me know if I can do anything to help.

**Comment by tgamblin** [comment]
@opadron this is ready to go so feel free to merge.

File: kubernetes/spack/slack-spack-io/web/deployments.yaml (hunk 1 above)
Comment (tgamblin) [reply kubernetes/spack/slack-spack-io/web/deployments.yaml:33]: It’s not. Even if fetched from a secret it would be sent to the client to post to google.

**Pull request merged**